.github/ISSUE_TEMPLATE/request-access.md

```diff
@@ -0,0 +1,12 @@
+## Content Area(s)
+If you have a dedicated content area, please check the corresponding box below. You can do this by either typing an "x" between the backets, or by first submitting this issue and then just clicking the box beside the content area.
+
+- [ ] Health Care
+- [ ] Disability
+- [ ] Education and Training
+- [ ] Careers and Employment
+- [ ] Pension
+- [ ] Housing Assistance
+- [ ] Life Insurance
+- [ ] Burials and Memorials
+- [ ] Records
```

Comment:
Is the purpose of this screen to get them to the right folder within the repo?

I kind of think it would simplify things to outline only one way to do it:
"Please type an "x" in the box that corresponds to your content area."

If they get familiar with github, they'll figure out on their own that there's a second way.

Reply:
That makes perfect sense!